### PR TITLE
Create BANNO.CHANGE.ADDR.V1.POW

### PR DESCRIPTION
**Pull request by tkainz**
BANNO.CHANGE.ADDR.V1.POW

### CODE REVIEW

File: change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW [hunk 1]
```diff
@@ -0,0 +1,1136 @@
...
+      END
+    END
+  END
+END [PROCEDURE]
+
+PROCEDURE PULLRGDATA
+[* Parse through the data passed back from the UI. Data will be delimited
+** with a pipe character.
+** PASS    @RGUSERCHR1  Street Address | Extra Address
+** PASS    @RGUSERCHR2  City | Stata | Zipcode
```
Comment (jucarson) [inline change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW:308]: Type Stata should be State

File: change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW [hunk 2]
```diff
@@ -0,0 +1,1136 @@
...
+   CURRBANNOID=UPPERCASE(DATALINEFIELD(1))
+   BNODEBUGMSG="RG4:"+CURRBANNOID
+   CALL BNODEBUGMSGADD
+   BNODEBUGMSG="BANNO ID:"+CURRBANNOID
+   CALL BNODEBUGMSGADD
+  END
+END [PROCEDURE]
+
+PROCEDURE CLEANUPTEXT
+[* Check character string and remove out any characters or hidden
```
Comment (jucarson) [inline change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW:387]: "remove out" should be simply"remove"

File: change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW [hunk 3]
```diff
@@ -0,0 +1,1136 @@
...
+      END
+    END
+  END
+ TMPCHR=TMPTEXT
+END [PROCEDURE]
+
+PROCEDURE VALIDATERGDATA
+[* Check the member's new street address to see if it contains
+** a PO Box address per the CU's configuration settings. If a
+** PO Box address be found, set an error code and create a
```
Comment (jucarson) [inline change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW:426]: "be found" should be "is found"

File: change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW [hunk 4]
```diff
@@ -0,0 +1,1136 @@
...
+**  SW     PARAMSETWARNINGS      Account level warning to set upon successful address change
+**  PR     PARAMPORESTRICT       Should the program check for and restrict a PO box entry?
+**  DB     DEBUGMODE             Debug mode on or off
+**
+** PASS    CONFIGFILENAME   Configuration Letter file name
+** RETURN  INVALIDACCTYPES()  Array of ineligible account types
+** RETURN  WARNINGSACCT()  Array of account warnings restricting access
+** RETURN  WARNINGTOCLEAR  Account warning to clear on successful addr. change
+** RETURN  WARNINGTOSET  Account warning to set on successful addr. change
+** RETURN  PARAMSETWARNINGEXPIRATION  Caldulated warning expiration date for WARNINGTOSET
```
Comment (jucarson) [inline change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW:610]: Caldulated should be Calculated

File: change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW [hunk 5]
```diff
@@ -0,0 +1,1136 @@
...
+
+  END
+END [PROCEDURE]
+
+PROCEDURE VALIDATECONFIG
+[* Validate some configuration Letter file values
+** PASS
+** RETURN  PARAMVALIDATIONERROR  Boolean flag
+** RETURN  PARAMVALIDATIONERRORCHR  Error return description
+** RETURN  RESPONSECODE  Error code pertinant to error type
```
Comment (jucarson) [inline change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW:786]: pertinant should be pertinent

**Review by jucarson — CHANGES_REQUESTED** [review]

File: change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW (hunk 1 above)
Comment (tkainz) [reply change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW]: Corrected

File: change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW (hunk 2 above)
Comment (tkainz) [reply change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW]: Corrected

File: change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW (hunk 3 above)
Comment (tkainz) [reply change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW]: Corrected

File: change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW (hunk 4 above)
Comment (tkainz) [reply change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW]: corrected

File: change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW (hunk 5 above)
Comment (tkainz) [reply change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW]: corrected

**Review by jucarson — APPROVED** [review]
Looks good!

**Pull request merged**